### PR TITLE
Add test to verify container metrics

### PR DESCRIPTION
**Pull request by ScarletTanager**
This commit checks that the container metrics have been submitted for the specified application.

[#113279137]

Signed-off-by: Sandy Cash <scarlet.tanager@gmail.com>

### CODE REVIEW

File: apps/loggregator_test.go [hunk 1]
```diff
@@ -113,6 +113,33 @@ var _ = Describe("loggregator", func() {
 			Expect(messages).To(ContainElement(EnvelopeContainingMessageLike("Muahaha")), "To enable the logging & metrics firehose feature, please ask your CF administrator to add the 'doppler.firehose' scope to your CF admin user.")
 		})
 	})
+
+	It(diegoUnsupportedTag+"shows container metrics", func() {
```
Comment (ematpl) [inline apps/loggregator_test.go]: I don't see why this test doesn't also apply to the Diego backend, as the cells also emit metrics about their running containers periodically.

**Comment by fraenkel** [comment]
It was more a statement that we didn't test it on Diego. If someone can verify there should be no reason why it wouldn't apply to both.

**Comment by Amit-PivotalLabs** [comment]
I agree with @ematpl, let's verify that this works on Diego and remove the tag.

**Comment by Amit-PivotalLabs** [comment]
@ScarletTanager @fraenkel are you folks able to verify this against Diego?  Should be as simple as focusing the test and running the test suite via ginkgo with `$CONFIG` set to a JSON file that specifies `"backend": "diego"` and targets an API with a Diego backend, e.g. BOSH-Lite or A1.

**Comment by fraenkel** [comment]
@Amit-PivotalLabs @ematpl Verified it works on Diego and DEA on A1. Commit has been updated to support both.

**Comment by robdimsdale** [comment]
Merged, thanks.

Rob && @dsabeti

**Pull request merged**